spec/meilisearch/index/settings_spec.rb

```diff
@@ -680,4 +682,51 @@ def update_synonyms(index, synonyms)
 
     client.wait_for_task(task['taskUid'])
   end
+
+  context 'On pagination sub-routes' do
+    let(:index) { client.index(uid) }
+    let(:pagination) { { maxTotalHits: 3141 } }
+
+    before { client.create_index!(uid) }
+
+    it 'gets default values of pagination' do
+      settings = index.pagination.transform_keys(&:to_sym)
+      expect(settings).to be_a(Hash)
+      expect(settings).to eq(default_pagination)
+    end
+
+    it 'updates pagination' do
+      task = index.update_pagination(pagination)
+      client.wait_for_task(task['taskUid'])
+      expect(task['type']).to eq('settingsUpdate')
+      expect(index.pagination.transform_keys(&:to_sym)).to eq(pagination)
+    end
```

Comment:
A good addition based here: http://blog.plataformatec.com.br/2014/04/improve-your-test-readability-using-the-xunit-structure/

```suggestion
    it 'updates pagination' do
      task = index.update_pagination(pagination)
      client.wait_for_task(task['taskUid'])

      expect(index.pagination.transform_keys(&:to_sym)).to eq(pagination)
    end
```